### ISSUE TITLE
Not able to start with not renewable token

### ISSUE THREAD

**Issue by d3m0n1c** [ricoberger/vault-secrets-operator#96]
Trying to run my project locally with not renewable token.
```
vault-secrets-operator:
  environmentVars:
    - name: VAULT_TOKEN
      value: "VaultRootToken123"
    - name: VAULT_TOKEN_LEASE_DURATION
      value: "86400"
```
Having following error:
```
{"level":"info","ts":1617181954.3451219,"logger":"vault","msg":"Reconciliation is enabled.","ReconciliationTime":0}
{"level":"info","ts":1617181954.345637,"logger":"vault","msg":"Renew Vault token"}
{"level":"error","ts":1617181954.3811648,"logger":"vault","msg":"Could not renew token","error":"Error making API request.\n\nURL: PUT http://host.docker.internal:8200/v1/auth/token/renew-self\nCode: 400. Errors:\n\n* lease is not renewable","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\t/go/pkg/mod/github.com/go-logr/zapr@v0.2.0/zapr.go:132\ngithub.com/ricoberger/vault-secrets-operator/vault.(*Client).RenewToken\n\t/workspace/vault/client.go:69"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xbebddd]

goroutine 12 [running]:
github.com/ricoberger/vault-secrets-operator/vault.(*Client).RenewToken(0xc0002e7110)
        /workspace/vault/client.go:72 +0x26d
created by main.main
        /workspace/main.go:60 +0x304
```
Can I somehow disable token renewing and work with a fixed token on Dev environment?

**Comment by ricoberger** [comment]
Hi @d3m0n1c, currently this is not possible. Maybe we can add another environment variable to disable token renewing.

**Comment by d3m0n1c** [comment]
@ricoberger, it would be great! Thanks.

**Comment by ricoberger** [comment]
Hi @d3m0n1c, instead of introducing a new environment variable, I fixed the nil pointer exception (#100).

The operator should now work with tokens, which can not be renewed. The renew function will still be triggered on a regular basis, but you can increase the interval with the `VAULT_TOKEN_RENEWAL_RETRY_INTERVAL` environment variable, e.g. to 1 hour `VAULT_TOKEN_RENEWAL_RETRY_INTERVAL=3600`.
